cmd/podman/root.go

```diff
@@ -363,6 +363,8 @@ func rootFlags(cmd *cobra.Command, opts *entities.PodmanConfig) {
 	lFlags.StringVarP(&opts.URI, "host", "H", uri, "Used for Docker compatibility")
 	_ = lFlags.MarkHidden("host")
 
+	lFlags.String("config", "", "Ignored for Docker compatibility")
```

Comment:
I don't like the user experience when silently ignoring the option.  I would expect it to have some impact.  An error as at the moment at least tells me that it's not going to work.

I would prefer to keep erroring out or do an `logrus.Errorf("--config is not supported")` that screams at the user.

Reply:
I can add the logrus.Error, but the problem was tools that were doing 
docker --config ...

When users commented out this option, everything worked properly.

If we start spewing errors then it could be an aggravation.

Reply:
I understand the intention.  It's a delicate art.  I am mostly afraid to let users believe into thinking that `--config $lifeOrDeath` would work.